sdk/eventhubs/azure-messaging-eventhubs/src/main/java/com/azure/messaging/eventhubs/EventHubProducerClient.java

```diff
@@ -202,7 +202,7 @@ void send(Iterable<EventData> events) {
      * @param events Events to send to the service.
      * @param options The set of options to consider when sending this batch.
      */
-    void send(Iterable<EventData> events, SendOptions options) {
+    public void send(Iterable<EventData> events, SendOptions options) {
```

Comment:
These need appropriate documentation of the exception that will be triggered, and any useful sample code.

Reply:
Updated documentation and added sample code.